Prepare release0.90.0

Andreas-Forster opened the pull request:
Updating the readme how to use the library with the new release. The changed informtaions assume that #135 is merged first.

Andreas-Forster committed:
updated README for release v0.90.0

BernhardEgger reviewed (approved):
only comment here would be that the developer list is a bit small. What if you leave? perhaps add some more adresses or a non-institutional email adress?
(I'm also fine with as is)

Andreas-Forster commented:
Thank you for your thoughtful comment. I changed the "developer" information now to our scalismo-faces community. This was done in PR #135 and will be merged before these changes. I just rebased this branch to the new state of the PR #135 .

Andreas-Forster commented:
Cleaned the PR in a bit of a complicated way. The changes are still the same introduced with this PR.
